internal/store/horizontalpodautoscaler.go

```diff
@@ -323,8 +323,8 @@ func hpaMetricFamilies(allowAnnotationsList, allowLabelsList []string) []generat
 
 					for _, m := range metrics {
 						metric := m
-						metric.LabelKeys = []string{"condition", "status"}
-						metric.LabelValues = append([]string{string(c.Type)}, metric.LabelValues...)
+						metric.LabelKeys = []string{"condition", "reason", "status"}
```

Comment:
nit: // Possible reasons are https://github.com/kubernetes/kube-state-metrics/issues/1299#issuecomment-725615247

Reply:
@voanhduy1512 could you please add this list to the doc?